index.js

```diff
@@ -72,7 +72,8 @@ function beautify(error, collection, values, messages) {
                 indexes[indexName].forEach(function (field) {
                     var path = field[0];
                     var props = {
-                        type: 'Duplicate value',
+                        type: 'unique',
+                        kind: 'unique',
```

Comment:
Will push and send you a message shortly @matteodelabre doing some tests with project using it atm, I added something else to add a message formatter to the string, will share in a bit